de.dlr.sc.virsat.uiengine.ui/src/de/dlr/sc/virsat/uiengine/ui/editor/snippets/general/UiSnippetActiveConcepts.java

```diff
@@ -134,6 +134,7 @@ public void widgetSelected(SelectionEvent e) {
 						// Create a map conceptName -> concept and add all concept names to the dependency tree
 						for (Object object : selectedObjects) {
 							Concept concept = ((ActiveConceptConfigurationElement) object).loadConceptFromPlugin();
+							EcoreUtil.resolveAll(concept);

```

Comment:
Why is the resolve all needed?
Wouldn't it make sense to move it into the load from the call above?